### PR TITLE
Enable linter on arm64

### PR DESCRIPTION
**Pull request by mathetake**
Signed-off-by: Takeshi Yoneda <takeshi@tetrate.io>

### CODE REVIEW

**Comment by mathetake** [comment]
huh maybe it's not possible to run golangci-lint for multi arch

File: Makefile [hunk 1]
```diff
@@ -37,9 +37,13 @@ build.spectest:
...
 	@go test ./...
 
+golangci_lint_goarch ?= $(shell go env GOARCH)
+gopath ?= $(shell go env GOPATH)
+
 .PHONY: lint
 lint:
-	@go run $(golangci_lint) run --timeout 5m
+	@go install $(golangci_lint)
+	@GOARCH=$(golangci_lint_goarch) $(gopath)/bin/golangci-lint run --timeout 5m
```
Comment (mathetake) [inline Makefile:46]: found out that `go run` cannot be combined with GOARCH modification

**Review by nullpo-head — APPROVED** [review]
The changes show the effectiveness of the lint!

**Comment by mathetake** [comment]
Thanks for the look!

**Pull request merged**